lookout/core/lib.py

```diff
@@ -75,18 +75,18 @@ def extract_changed_nodes(root: Node, lines: Sequence[int]) -> List[Node]:
     return result
 
 
-def files_by_language(files: Iterable[File]) -> Dict[str, Dict[str, File]]:
+def files_by_language(files: Iterable[File]) -> Dict[str, Iterable[File]]:
     """
     Sorts files by programming language and path.
 
     :param files: Iterable of `File`-s.
-    :return: Dictionary with languages as keys and files mapped to paths as values.
+    :return: Dictionary with languages as keys and sorted files by path as values.
     """
-    result = defaultdict(dict)
-    for file in files:
+    result = defaultdict(list)
```

Comment:
Actually, we can use `OrderedDict` from `collections` and I think it is worth to use it for language level also:
```python
result = OrderedDict()  # type: OrderedDict[str, OrderedDict[str, str]]
```

this way we avoid changing the logic in style-analyzer.